### ISSUE TITLE
Introduction to "dataclasses" in the standard library

### ISSUE THREAD

**Issue by cjrh** [BrisPy/talks#39]

**Comment by rancoxu** [comment]
Hi Ryan just wondering is this section about using numpy/pandas/sklearn/matplotlib and other data science packages? in that case i'd love to take this slot.
Thanks

**Comment by kaiw** [comment]
This talk topic was actually focusing on the dataclasses module in the standard library.

Having said that, I think that an introduction to some of the more widely used data science packages is also a good idea! If we don't have an issue open for this, feel free to propose it as a talk.

**Comment by rancoxu** [comment]
yeah why not!? i do have experience in tutoring data science packages in python. It'll be fun to share with more like-minded people. how about an introduction about numpy/pandas as a kick-off?

**Comment by kaiw** [comment]
@rancoxu I've created #46, so let's continue discussion over there.
